uktrade/icms
# ICMSLST-2694 - Country translations

Adds translations for new countries added 

To test: 
 -  switch to branch
 -  run `python manage.py check_data_counts --checks counts` Note the failures for translations
 -  run `python manage.py add_v2_reference_data`
 - rerun check command `python manage.py check_data_counts --checks counts`
 - Check the translations view `http://caseworker:8080/country/translations/3/edit/`
